docs/sp-add-ins/publish-sharepoint-add-ins.md

```diff
@@ -1,33 +1,29 @@
 
-- **An internal organization add-in catalog** Publish your add-in to an internal organization add-in catalog, hosted on your SharePoint deployment, to make it available to users who access that SharePoint deployment.
+- **An internal organization add-in catalog** - Publish your add-in to an internal organization add-in catalog, hosted on your SharePoint deployment, to make it available to users who access that SharePoint deployment.
 
 For information about how to package your add-in for publication by using Visual Studio 2012, see [Publish SharePoint Add-ins by using Visual Studio](publish-sharepoint-add-ins-by-using-visual-studio.md).
 
 ## Publishing to AppSource
 
-To publish an add-in to AppSource, you must first [register as a Microsoft developer](https://developer.microsoft.com/en-us/store/register). 
+To publish an add-in to AppSource, you must first [open a developer account](docs.microsoft.com/office/dev/store/open-a-developer-account). 
 
-When you upload an add-in to AppSource for publication, there is an approval process that Microsoft performs. For example, it checks that the add-in is free of viruses, whether the add-in manifest markup is valid and complete and verifies that any SharePoint solution packages (.wsp files) that you included in the add-in do not contain elements that aren't allowed, or SharePoint features with a scope that is broader than web. The package is also inspected for objectionable content. If the add-in package passes verification, it's wrapped into a file and signed by Microsoft.
+When you upload an add-in to AppSource for publication, Microsoft performs a validation check. For example, it checks that the add-in is free of viruses and that the add-in manifest markup is valid and complete, and verifies that any SharePoint solution packages (.wsp files) that you included in the add-in do not contain elements that aren't allowed, or SharePoint features with a scope that is broader than web. The package is also inspected for objectionable content. If the add-in package passes validation, it's wrapped into a file and signed by Microsoft.
 
-When you upload your add-in for publication on AppSource, you can choose the terms of the license you want to offer users when they download it. Use this add-in license to decide: 
-
-- Whether you are offering your add-in for free, trial, or for purchase.
-- Whether your add-in can be acquired on a per-user or site basis.
-
-SharePoint does not enforce license terms for add-in usage. Instead, it provides a licensing framework that lets you include code logic in your add-in to enforce whatever licensing restrictions you choose. For example, you can include code logic in your add-in that enables users to access certain add-in features if they have a paid license, but not if they have a trial license. For more information, see [License your Office and SharePoint Add-ins](https://docs.microsoft.com/en-us/office/dev/store/license-your-add-ins).
+> [!NOTE] 
```

Comment:
FYI, the licensing model for add-ins is going to be retired; this is the content we'll be publishing for the Partner Center GA.